import_package.md

```diff
@@ -0,0 +1,280 @@
+# Import package
+
+* An import package is a .zip file containing a **manifest.yaml** that describes additional 
+  contents and actions that should be carried out when processing the package.
+* A valid import package must contain all resource files that are referenced in the 
```

Comment:
```suggestion
An import package is a .zip file containing a **manifest.yaml** that describes additional 
  contents and actions that should be carried out when processing the package.
A valid import package must contain all resource files that are referenced in the 
```
I'd make this straight text rather than bullets